src/protocol_nmea.cpp

```diff
@@ -399,9 +400,8 @@ void nmea_enable_stream(rmci_t &rmci, uint32_t nmeaId, uint8_t periodMultiple)
 	case NMEA_MSG_ID_RMC:       
 	case NMEA_MSG_ID_ZDA:       
 	case NMEA_MSG_ID_PASHR:     
-	case NMEA_MSG_ID_PSTRB:     
-	case NMEA_MSG_ID_INFO:      
-	case NMEA_MSG_ID_GSV:       did = DID_GPS1_POS; break;	
+	case NMEA_MSG_ID_PSTRB:     did = DID_GPS1_POS; break;	
+	case NMEA_MSG_ID_GSV:       did = DID_GPS1_SAT; break;	
```

Comment:
Subtle!   SAT not POS!

Reply:
Yes.  And the NMEA_MSG_INFO is associated with DID_DEV_INFO, not DID_GPS1_POS.  Two mistakes.